alf/utils/value_ops.py

```diff
@@ -255,3 +255,36 @@ def generalized_advantage_estimation(rewards,
         advs = advs.transpose(0, 1)
 
     return advs.detach()
+####### add for the retrace method
+def generalized_advantage_estimation_retrace(importance_ratio, discounts, rewards, td_lambda, time_major, values, target_value,step_types):
```

Comment:
please comment following the way of other functions.

Reply:
Also need unittest for this function.

Reply:
1) line too long
2) add space after ```,```
3)  comments for the function need to be added